# Generated for NaoQi method ALBehaviorManager.stopBehavior
# Stop a behavior

# The behavior directory name
string name
---


